python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -515,18 +509,21 @@ class RandomForestClassifier(Base):
            to a shared file. Cuml issue #1854 has been created to track this.
     """
     def _tl_model_handles(self, model_bytes):
-        cdef ModelHandle cuml_model_ptr = NULL
+        cdef uintptr_t tl_handle_int
```

Comment:
In another pr, I proposed renaming this something like `_alloc_and_convert_model` to make it clear that the caller needs to free the result.